src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/monitoring/IncompleteEnvelopesDisabledTaskTest.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.tasks.monitoring;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.ApplicationContext;
+import org.springframework.test.context.junit4.SpringRunner;
+import uk.gov.hmcts.reform.bulkscanprocessor.config.IntegrationTest;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@IntegrationTest
+@RunWith(SpringRunner.class)
+public class IncompleteEnvelopesDisabledTaskTest {
```

Comment:
do we need this?

Reply:
same as discussed previously in hmcts/bulk-scan-processor#601 ([specific comment](https://github.com/hmcts/bulk-scan-processor/pull/601#discussion-diff-272147812)). each flag we have we could "assure" it works with such kind of test. perhaps even a bundle in one